econml/utilities.py

```diff
@@ -1577,3 +1577,40 @@ def transpose_dictionary(d):
         for key2, val in value.items():
             output[key2][key1] = val
     return output
+
+
+class RegWrapper:
+    """
```

Comment:
I think `RegressionWrapper` would be more appropriate for a public component.

Reply:
Should this also pass attributes through to the underlying classifier?

Reply:
I was having a problem with cloning if I enable that and for now we don't need it, it's just for internal use. If we use it further we can enable this in a future PR.

Reply:
I changed the name

Reply:
If it's just for internal use, put an underscore at the beginning so it doesn't end up in the public docs.